CHANGELOG

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [0.2.9] - 2020-03-25
+### Bug Fixes
+ - Fix verification errors due to outdated values. https://github.com/nmstate/nmstate/issues/853
```

Comment:
IMHO it would be nicer to use a colon:

```suggestion
 - Fix verification errors due to outdated values: https://github.com/nmstate/nmstate/issues/853
```

Reply:
Do you prefer the same for known issues?

Reply:
Done in both places, thanks!